tools/hotplug/Linux/systemd/xen-pcid.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Xen-pcid - run xenpcid daemon
+Requires=xenstored.service
+After=xendomains.service
+ConditionPathExists=/proc/xen/capabilities
+
+[Service]
+Type=forking
+EnvironmentFile=-@CONFIG_DIR@/@CONFIG_LEAF_DIR@/xencommons
+ExecStart=@sbindir@/xenpcid
+KillSignal=USR1
```

Comment:
According to this, server should be stopped with `SIGUSR1` signal, but I can't see a signal handler anywhere in the code.